src/lib/parser/resolveCircularRef.ts

```
@@ -48,13 +49,26 @@
   }
 }
 
-function detectCircularReference(ancestor: SchemaNode | null, value: any): string | null {
+function detectCircularReference(
+  ancestor: SchemaNode | null,
+  value: any
+): string | null {
```

ESLint

[error] Missing trailing comma.

(style/comma-dangle)